pymatviz/utils.py

```diff
@@ -362,6 +370,59 @@ def si_fmt(
 si_fmt_int = partial(si_fmt, fmt=".0f")
 
 
+def get_cbar_label_formatter(
```

Comment:
are you sure we need to make this a function? might make sense to inline this code since it seems to only be used one and the code isn't that complex

Reply:
I think the reason I made some function that could be done inline is that I really want them to be well covered by unit tests :)

I have no objection to make this inline, but I reaaaaally don't want to lose those unit tests. I would appreciate it if you could give me some advice on this :)

And I'm hoping this could be useful for other plotters which include a colorbar.